rotkehlchen/data_import/importers/rotki_trades.py

```diff
@@ -39,6 +39,12 @@ def _consume_rotki_trades(
         amount_sold = deserialize_asset_amount(csv_row['Sell Amount'])
         amount_bought = deserialize_asset_amount(csv_row['Buy Amount'])
         asset, fee, fee_currency, location, timestamp = process_rotki_generic_import_csv_fields(csv_row, 'Base Currency')  # noqa: E501
+        if csv_row['Type'].strip().lower() == 'buy':
+            trade_type = TradeType.BUY
+        elif csv_row['Type'].strip().lower() == 'sell':
+            trade_type = TradeType.SELL
+        else:
+            raise InputError(f'Unknown trade type {csv_row["Type"]}')
```

Comment:
The docstring is not updated to add this error but I believe it should be DeserializationError instead